tests/base_task_test.py

```diff
@@ -0,0 +1,223 @@
+import pytest
+
+from dbt_sugar.core.task.base import COLUMN_NOT_DOCUMENTED
+from dbt_sugar.core.task.doc import DocumentationTask
+
+
+def __init_descriptions():
+    base_task = DocumentationTask(None)
```

Comment:
Not sure why you call this `base_task` since you're testing the documentation task I guess you could use this as the variable name no?

Reply:
On this one I would still rename `base_task` to `documentation_task` or `doc_task` to avoid people (or us wondering why).